### ISSUE TITLE
Show roads on map

### ISSUE THREAD

**Issue by CharonM72** [CharonM72/SatelliteMapMaker#7]

**Comment by PeridexisErrant** [comment]
I'd actually leave them out - the roads are way sub-pixel size on these maps, and it could look pretty strange on older maps with a lot of them.  A flag for roads/no roads would be a good cover for that though!

**Comment by CharonM72** [comment]
Yeah I'll have to see how they look before putting them in. Since this is supposed to represent a satellite image, I was planning to just cut roads out of the vegetation layers and leave the dirt layer for dirt roads (or slightly darken/lighten) and make grey lines for stone roads. If it looks ugly I'll remove it. Rivers looked really ugly at first too, but lowering their opacity a lot greatly improved it. Blur can also help.
